docs/DesignDocs/SGX_Endorsements_V2.md

```diff
@@ -0,0 +1,222 @@
+SGX Attestation Endorsements V2 Updates
+================
+
+Intel SGX released v2 of the Provision Certification Service for ECDSA Attestation.  The
+following APIs were affected:
+
+1. Get TCB Info.
+2. Get Quoting Enclave Identity.
+
+[API Version 1](https://api.portal.trustedservices.intel.com/documentation#pcs-certificate)
+
+[API Version 2](https://api.portal.trustedservices.intel.com/documentation#pcs-certificate-v2)
+
+This document describes the changes for supporting version 2 of the API as well as keeping
+the existing support for version 1 of the API.  This document assumes the user is familiar with
+the SGX attestation endorsements and how it is used during SGX quote verification.
+
+Motivation
+----------
+
+This change is required for customers who would like to use the v2 API web endpoints. Note that the Azure PCK
+Caching Service and the Azure DCAP Client will also need to be updated and it is outside of the this specification.
+
+User Experience
+---------------
+
+The user experience does not change, it should be seamless to the user.  The caller will be able to identify the version of the data by checking the existing `version` field.
+
+The quote verifier will specify whether to use the v1 or v2 Azure PCK Caching Service web API endpoints.  The
```

Comment:
Clarify: the text here mentions Azure... is this design doc only usable with Azure?  If so, I'd recommend changing the title.  If not, I'd recommend changing the text here to clarify.

Reply:
No it's not only usable with Azure.

Reply:
The title still says "SGX Attestation Endorsements V2 Updates", nothing with Azure in the title.  Your response indicated that this doc was only relevant to people who use Azure, not on-prem SGX, so the title still needs updating unless I'm missing something.

Reply:
Updated title. Thank you Dave.